composer.json

```diff
@@ -49,7 +49,8 @@
         "squizlabs/php_codesniffer": "3.7.1",
         "symfony/cache": "^5.4|^6.0",
         "symfony/console": "^4.4|^5.4|^6.0",
-        "vimeo/psalm": "4.27.0"
+        "vimeo/psalm": "4.27.0",
+        "ext-pcntl": "*"
```

Comment:
Agh... it's unavailable on Windows. In this case, a PHPUnit test case annotation is due instead. See the following for example: https://github.com/doctrine/dbal/blob/2ab785585ed257717760eb6ddcd12bfb96f97639/tests/Functional/Driver/IBMDB2/StatementTest.php#L15-L16